pl_engine/pl_server/src/main/java/com/cubrid/plcsql/compiler/ast/StmtSql.java

```diff
@@ -31,11 +31,34 @@
 package com.cubrid.plcsql.compiler.ast;
 
 import com.cubrid.plcsql.compiler.Coercion;
+import com.cubrid.plcsql.compiler.ast.loopOpt.SqlUse;
 import com.cubrid.plcsql.compiler.type.Type;
 import java.util.List;
 import org.antlr.v4.runtime.ParserRuleContext;
 
-public abstract class StmtSql extends Stmt {
+public abstract class StmtSql extends Stmt implements SqlUse {
+
+    public StmtLoop containerLoop;
```

Comment:
```suggestion
    private StmtLoop containerLoop;
```
setter가 존재하므로 getter도 추가하여, public 보다는 private으로 선언하여 관리를 용이하게 하는 것이 좋을 것 같습니다.